assets/voyager/src/index.js

```diff
@@ -9,7 +9,7 @@ import jQuery from 'jquery';
  */
 Drupal.behaviors.graphQLRenderVoyager = {
   attach: (context, settings) => {
-    const container = jQuery('#graphql-voyager', context).once('graphql-voyager')[0] || undefined;
+    const container = jQuery(once('graphql-voyager', '#graphql-voyager', context))[0] || undefined;
```

Comment:
I believe this needs to be imported from the `once` package? I don't know how the JavaScript build goes here. I think `asset/voyager` will need `@drupal/once` and import `once` from there. Maybe?

Reply:
Added the [@drupal/once](https://www.npmjs.com/package/@drupal/once) package to voyager in commit [6ec7c41](https://github.com/drupal-graphql/graphql/commit/6ec7c417aae80d8804a9179527e9d079044c94ad)